ambry-store/src/test/java/com.github.ambry.store/StorageManagerTest.java

```diff
@@ -163,7 +230,7 @@ public void shutdownBlobStoreTest() throws Exception {
     List<? extends ReplicaId> invalidPartitionReplicas = invalidPartition.getReplicaIds();
     StorageManager storageManager = createStorageManager(replicas, metricRegistry);
     storageManager.start();
-    for (int i = 0; i < replicas.size() - 1; i++) {
+    for (int i = 1; i < replicas.size() - 1; i++) {
```

Comment:
why the change from 0 to 1?

Reply:
First store is used to test a special case where we try to stop a store when diskManager is not running.